todos/src/components/LoadMoreBtn.js

```diff
@@ -1,10 +1,34 @@
 import React from "react";
 import "styles/loadmoreBtn.css";
+import { useSelector, useDispatch } from "react-redux";
+import { updateCurrentPage } from "actions";
+
+function LoadMoreBtn(props) {
+  const dispatch = useDispatch();
+  const currentPage = useSelector(
+    (state) => state.currentPageReducer.currentPage
+  );
+
+  const handleLoadMore = () => {
+    dispatch(updateCurrentPage(currentPage + 1));
+  };
+
+  const handleshowLess = () => {
+    dispatch(updateCurrentPage(1));
+  };
 
-function LoadMoreBtn() {
   return (
     <div className="load-more">
-      <button className="load-more__button">Load More</button>
+      {props.type === "Load More" && (
```

Comment:
Please maintain a constants.js file to host variables that doesn't change that much. 

```js
import {LOAD_MORE} from "utils/constants.js"
{props.type ===  LOAD_MORE &&
```